backend/endpoints/insert.py

```
@@ -23,14 +33,104 @@ def add_to_assembly(**kwargs):
+    def __init__(
+        self,
+        name: str,
+        part_studio_to_add: ElementPath,
+        microversion_id: str,
+        configuration: dict | None = None,
+        parameters: dict | None = None,
+    ):
+        self.name = name
+        self.namespace = path_to_namespace(part_studio_to_add, microversion_id)
+
+        if configuration != None and parameters != None:
```

Ruff (0.12.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)